packages/api/endpoints/elasticsearch.js

```diff
@@ -213,6 +213,8 @@ async function indexFromDatabase(req, res) {
       asyncOperationTaskDefinition: process.env.AsyncOperationTaskDefinition,
       cluster: process.env.EcsCluster,
       lambdaName: process.env.IndexFromDatabaseLambda,
+      description: 'Elasticsearch Reindex',
```

Comment:
I was looking through the code and this is actually an Elasticsearch index from database where we populate ES with what we have in the DB.

Reply:
Changed